icons/yelp.svg

```diff
@@ -1 +1 @@
.org/2000/svg"><title>Yelp</title><path d="m8.7 15.143 1.106-.256c.023-.005.062-.014.108-.029a1.22 1.22 0 0 0 .872-1.452l-.004-.02a1.215 1.215 0 0 0-.188-.411 1.539 1.539 0 0 0-.45-.387 4.172 4.172 0 0 0-.642-.297l-1.212-.442c-.68-.253-1.36-.5-2.046-.74-.445-.158-.822-.297-1.15-.399-.062-.019-.13-.038-.186-.057-.396-.121-.674-.172-.91-.173a1.084 1.084 0 0 0-.459.083 1.174 1.174 0 0 0-.399.283c-.055.064-.107.13-.156.198a2.332 2.332 0 0 0-.233.464 6.287 6.287 0 0 0-.331 2.064c.004.634.02 1.449.37 2 .084.142.196.265.33.36.249.171.5.194.761.213.39.028.768-.068 1.145-.155l3.672-.848H8.7zm12.33-5.868a6.278 6.278 0 0 0-1.2-1.71 1.846 1.846 0 0 0-.204-.175 2.398 2.398 0 0 0-.208-.14 2.358 2.358 0 0 0-.227-.109 1.17 1.17 0 0 0-.482-.08c-.157.01-.31.053-.45.127-.21.105-.438.273-.742.555-.042.042-.094.089-.142.133-.25.235-.528.525-.86.863-.511.517-1.015 1.037-1.516 1.563l-.896.93c-.164.17-.314.352-.447.548-.113.164-.194.35-.236.545-.025.15-.021.304.01.452l.005.02a1.218 1.218 0 0 0 1.418.923.947.947 0 0 0 .11-.021l4.779-1.104c.377-.087.759-.167 1.097-.363.227-.132.443-.262.59-.525.08-.145.128-.304.141-.469.074-.65-.266-1.39-.54-1.963zm-8.552 2.01c.346-.435.345-1.084.376-1.614.105-1.77.214-3.542.301-5.314.034-.671.106-1.333.066-2.01-.033-.557-.037-1.198-.39-1.656-.621-.807-1.947-.74-2.852-.616-.277.039-.554.09-.83.157s-.548.138-.814.223c-.868.285-2.089.807-2.295 1.807-.117.565.16 1.144.373 1.66.26.625.615 1.189.938 1.778.854 1.554 1.724 3.099 2.592 4.645.26.462.542 1.047 1.043 1.286.033.014.067.027.101.038.225.085.47.102.704.047l.042-.01c.217-.058.412-.176.566-.34.028-.025.054-.053.079-.081zm-.414 4.741a1.105 1.105 0 0 0-1.231-.415 1.136 1.136 0 0 0-.153.064 1.484 1.484 0 0 0-.217.135 2.91 2.91 0 0 0-.52.532c-.038.049-.073.114-.12.156l-.768 1.057c-.436.592-.865 1.186-1.291 1.79-.278.389-.519.718-.709 1.008-.036.055-.073.116-.107.165-.228.352-.357.61-.423.838-.05.152-.065.314-.046.472.021.166.077.325.164.467.046.072.095.141.148.207a2.34 2.34 0 0 0 .386.356c.531.37 1.112.634 1.723.84a6.133 6.133 0 0 0 1.571.3 2.415 2.415 0 0 0 .523-.041c.083-.02.165-.044.245-.072a1.1 1.1 0 0 0 .672-.665c.089-.22.147-.502.186-.92.003-.059.011-.13.017-.195.03-.346.044-.753.067-1.231.037-.736.067-1.469.09-2.203l.05-1.306c.01-.3.002-.634-.082-.934a1.397 1.397 0 0 0-.175-.405zm8.675 2.044c-.16-.176-.388-.352-.746-.568l-.169-.101c-.298-.18-.658-.369-1.078-.597-.645-.354-1.29-.7-1.942-1.042l-1.152-.61c-.06-.018-.12-.061-.177-.088a2.863 2.863 0 0 0-.699-.25 1.481 1.481 0 0 0-.254-.026 1.016 1.016 0 0 0-.165.01 1.107 1.107 0 0 0-.923.913c-.017.146-.012.294.016.439.056.306.193.61.335.875l.615 1.152c.342.65.688 1.297 1.043 1.94.23.421.42.78.598 1.079.034.056.073.116.102.168.217.359.391.584.569.746a1.1 1.1 0 0 0 .894.302 2.4 2.4 0 0 0 .251-.044c.082-.022.162-.048.241-.078.085-.032.168-.07.248-.115a6.097 6.097 0 0 0 1.281-.96 6.73 6.73 0 0 0 1.182-1.51c.044-.08.082-.162.114-.248a2.5 2.5 0 0 0 .077-.24c.02-.084.034-.168.043-.252a1.19 1.19 0 0 0-.056-.491 1.093 1.093 0 0 0-.248-.404z"/></svg>
```

Comment:
As mentioned in #9072 and according to https://www.yelp.com/brand, I'm wondering if we need to add the ® symbol.

> Yelp branding should at all times be accompanied by the ® symbol.

Reply:
The R symbol is at the wordmark not the logo. How do we include it?

Reply:
The Zip file containing logos found at https://www.yelp.com/brand includes the ® symbol in the file `yelp_logos/Burst/yelp_burst.svg`

Reply:
Aplogies for the approval below - I wasn't aware this discussion was ongoing. Given they're explicit in their rules - then yes I think we should include the ®️.

Reply:
@service-paradis I missed this one. Will use it to update PR. Thanks ☺️

Reply:
Fixed @adamrusted